LICENSE-binary

```diff
@@ -261,7 +261,7 @@ io.grpc:grpc-protobuf:1.26.0
 io.grpc:grpc-protobuf-lite:1.26.0
 io.grpc:grpc-stub:1.26.0
 io.netty:netty:3.10.6.Final
-io.netty:netty-all:4.1.42.Final
+io.netty:netty-all:4.1.72.Final
 io.netty:netty-buffer:4.1.27.Final
```

Comment:
How about updating the version to 4.1.72 in other netty4 libraries?

Reply:
Thanks for review. Almost forgot.